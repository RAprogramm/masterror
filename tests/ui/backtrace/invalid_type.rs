// SPDX-FileCopyrightText: 2025 RAprogramm <andrey.rozanov.vl@gmail.com>
//
// SPDX-License-Identifier: MIT

use masterror::Error;

#[derive(Debug, Error)]
#[error("invalid backtrace field")]
struct InvalidBacktrace {
    #[backtrace]
    trace: String
}

fn main() {}
