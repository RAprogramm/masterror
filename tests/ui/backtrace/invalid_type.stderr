error: fields with #[backtrace] must be std::backtrace::Backtrace or Option<std::backtrace::Backtrace>
  --> tests/ui/backtrace/invalid_type.rs:10:5
   |
10 |     #[backtrace]
   |     ^^^^^^^^^^^^
